src/benchmarklib/benchmark_runner.cpp

```diff
@@ -198,9 +199,14 @@ void BenchmarkRunner::_benchmark_ordered() {
 
     const auto& name = _benchmark_item_runner->item_name(item_id);
     std::cout << "- Benchmarking " << name << std::endl;
-
+    // clear results
+    _results[item_id].successful_runs = {};
+    _results[item_id].unsuccessful_runs = {};
+    _results[item_id].duration = {};
```

Comment:
This is temporary just to be able to execute an item more than once, correct? It will falsify the results reported by the benchmark.

Reply:
Exactly